builds/jazz-build-module/sls-app/serverless-build-rules.yml

```diff
@@ -477,15 +484,20 @@ package: # Optional deployment packaging configuration
     sbr-render: user-only
     sbr-value:
       sbr-default: false
-      sbr-examples: false # Config if Serverless should automatically exclude dev dependencies in the deployment package. Defaults to true
+      sbr-example: false # Config if Serverless should automatically exclude dev dependencies in the deployment package. Defaults to true
 
 functions:
     sbr-type: "[aws-var-name:function]"
     sbr-render: user-only
 
+# works as expected when using sbr-formula only NOT with sbr-example
 plugins:
-    sbr-type: "[aws-var-name:plugin]"
-    sbr-render: user-only
+    sbr-type: "[plugin]"
+    sbr-render: merge
+    sbr-mandatory: true
+    sbr-value:
+      sbr-formula:
+       - serverless-plugin-log-subscription
```

Comment:
Quick question - What if platform needs to inject multiple plugins by default?

Reply:
Then we would need to provide that list here.